spec/example-unnamed-annotation.ttl

```diff
@@ -0,0 +1,4 @@
+PREFIX prov: <http://www.w3.org/ns/prov#>
+
```

Comment:
missing:
```
@prefix foaf: <http://xmlns.com/foaf/0.1/> .
@prefix dcterms: <http://purl.org/dc/terms/> .
```

Reply:
I followed the _somewhat_ consistent pattern of not repeating previously declared prefixes. (My plan is to extract all examples to files and use `data-include` once this PR lands. Then it's easy enough to validate even when they're "fragments", i.e. hoist prefixes with a `g/^PREFIX /m0`. Or perhaps have a "prelude" before the examples.)

Reply:
This makes sense in the future, but for now it's worth sticking to some clear rule. The fact that half of the prefixes are declared and the other half are not may seem strange to the reader.

Reply:
Yes. We should stick to one of A) consistently follow the current pattern of relying on previously declared prefixes and declare new ones as they're introduced; B) keep self-contained examples, or C) declare all prefixes to be used in a separate, initial prelude.

B is verbose but clear; examples can be copied as-is. (I think the Turtle spec follows this. I'll update to that unless anyone objects.)

Reply:
Having looked at the options, let's address this in a subsequent PR; to update all examples consistently.

Reply:
I added https://github.com/w3c/rdf-primer/issues/18 to track this. (Considering this conversation resolved with that.)